protected/humhub/modules/user/views/profile/about.php

```diff
@@ -12,20 +12,32 @@
 $categories = $user->profile->getProfileFieldCategories();
 ?>
 <div class="panel panel-default">
-    <div
-        class="panel-heading"><?= Yii::t('UserModule.views_profile_about', '<strong>About</strong> this user'); ?></div>
+
+    <div class="panel-heading">
+        <?= \Yii::t('UserModule.views_profile_about', '<strong>About</strong> this user'); ?>
+    </div>
+
     <div class="panel-body">
+
+        <p>
+            The member of <?= \Yii::$app->name ?> since date <?= Yii::$app->formatter->asDate($user->created_at) ?>
```

Comment:
Instead of using `The` use `A` better grammar, or even use `A proud member of <?= \Yii::$app->name ?> since date <?= Yii::$app->formatter->asDate($user->created_at) ?>`. :smile: